integration-test/helpers/pages/loginPage.js

```diff
@@ -66,11 +67,15 @@ export function loginPage (page, opts = {}) {
         }
 
         /**
-         * @param {string} username
+         * @param {string} text
          * @return {Promise<void>}
          */
-        async assertTooltipNotOpen (username) {
-            await expect(page.locator(`button:has-text("${username}")`)).not.toBeVisible()
+        async assertTooltipNotOpen (text) {
+            await expect(page.locator(`button:has-text("${text}")`)).not.toBeVisible()
+        }
+
+        async assertTooltipOpen (text) {
```

Comment:
Worth adding the type?

```suggestion
        /**
         * @param {string} username
         * @param {string} text
         * @return {Promise<void>}
         */
        async assertTooltipOpen (text) {
```

Reply:
Added https://github.com/duckduckgo/duckduckgo-autofill/pull/631/commits/03366ad416641e15bf246cddafef8d9b5cbee473